server.lua

```diff
@@ -89,7 +107,19 @@ end
 RegisterNetEvent('customs:server:saveVehicleProps', function()
     local src = source --[[@as number]]
     local vehicleProps = lib.callback.await('customs:client:vehicleProps', src)
+    currentAdmins[src] = currentAdmins[src] or {}
+    currentAdmins[src].admin = false
     if IsVehicleOwned(vehicleProps.plate) then
         MySQL.update.await('UPDATE player_vehicles SET mods = ? WHERE plate = ?', {json.encode(vehicleProps), vehicleProps.plate})
     end
 end)
+
+--Commands
+lib.addCommand('admincustoms', {
+    help = 'Toggle customs menu for admins',
+    restricted = 'group.admin',

```

Comment:
Should this not be `'customs.admin'`?
```suggestion
    restricted = 'customs.admin',
```